### PR TITLE
CFY-7859 Improve cfy_manager help messages

### PR DESCRIPTION
**Pull request by mcouthon**

### CODE REVIEW

File: packaging/install.sh [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
-echo "(members of group wheel have write access)"
-echo "###################################################################"
+echo "
+###########################################################################
+Cloudify installer is ready!
+To use it run:
+cfy_manager install --private-ip <PRIVATE_IP> --public-ip <PUBLIC_IP>
+(Use cfy_manager -h for a full list of options)
+
+Alternatively, edit /etc/cloudify/config.yaml to set IPs and other options,
```
Comment (bmansheim) [inline packaging/install.sh]: Replace line with: You can specify the public and private IP addresses as command options or in /etc/cloudify/config.yaml. You can also specify other installation settings in the config.yaml file.
If you specify the public and private IP addresses in the config.yaml file, run:
cfy_manager install

Comment (mcouthon) [reply packaging/install.sh]: Actually, that sounds too verbose, and even a bit more confusing than the original version. Maybe something in the middle?

Comment (mcouthon) [reply packaging/install.sh]: @bmansheim ?

Comment (bmansheim) [reply packaging/install.sh]: You can specify more installation settings in the config.yaml file. If you specify the public and private IP addresses in the config.yaml file, run:
cfy_manager install

**Pull request merged**